src/Core/Prefab/ColorLayersOrdering.js

```diff
@@ -0,0 +1,72 @@
+import { ImageryLayers } from '../Layer/Layer';
```

Comment:
I'd move this file either in `../Layer` or in `../../Rendering/` (because it's heavily related to `LayeredMaterial`)